front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -48,9 +49,21 @@ const ConsistCardTitle = ({
 export type StdcmConsistProps = {
   isDebugMode: boolean;
   disabled?: boolean;
+  setMissingMandatoryFields?: (
```

Comment:
This is always defined (you can also remove the `if` l.156)

```suggestion
  setMissingMandatoryFields: (
```

Reply:
It's now deleted